### DISCUSSION TITLE
Change the URL style to that of Gfycat

### DISCUSSION THREAD

**Discussion by Smallinger** [diced/zipline#322] [Ideas]
Would be cool if you can set in the admin panel the URL style like this one from Gfycat.
With some conditions such as how many adjectives.

here is a example
YOUR URL/adventurous-deadly-ape

here some lists.
[adjectives.txt](https://github.com/diced/zipline/files/10949528/adjectives.txt)
[animals.txt](https://github.com/diced/zipline/files/10949529/animals.txt)

**Comment by diced** [comment]
Added in https://github.com/diced/zipline/commit/f8cd847588f5bb471befc07a8095b0d99d8f6948
